src/Microsoft.CorrelationVector.UnitTests/CorrelationVectorTests.cs

```diff
@@ -164,12 +164,12 @@ public void SimpleIncrementCorrelationVectorTest()
         [TestMethod]
         public void SimpleExtendCorrelationVectorTest()
         {
-            var correlationVector = new CorrelationVector();
+            var correlationVector = new CorrelationVectorV1();
             var splitVector = correlationVector.Value.Split('.');
             var vectorBase = splitVector[0];
             var extension = splitVector[1];
 
-            correlationVector = CorrelationVector.Extend(correlationVector.Value);
+            correlationVector = CorrelationVectorV1.Extend(correlationVector.Value);
```

Comment:
In general, whenever we are using static methods in the tests, we should be using CorrelationVector not the explicit version (unless we are testing the explicit version)

Reply:
The problem here is that I can't actually create an instance of an abstract class in C#.